index.html

```diff
@@ -7,7 +7,6 @@
   </head>
   <body>
     <div id="root"></div>
-    <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
@imrishabh18 i've adapted this repo to just always use the hook and never the cdn, the start script now watches the build